CONTRIBUTING.md

```diff
@@ -0,0 +1,97 @@
+[![Open Source Love](https://badges.frapsoft.com/os/v1/open-source.png?v=103)](https://github.com/ellerbrock/open-source-badges/)
+
+- Take a look at the Existing [Issues](https://github.com/eddiejaoude/stargate) or create your own Issues!
+- Wait for the Issue to be assigned to you after which you can start working on it.
+- Fork the Repo and create a Branch for any Issue that you are working upon.
+- Create a Pull Request which will be promptly reviewed and suggestions would be added to improve it.
+- Add Screenshots to help us know what this Script is all about.
+
+
+## ⭐HOW TO MAKE A PULL REQUEST:
+
+**1.** Fork [this](https://github.com/eddiejaoude/stargate) repository.
+Click on the <a href="https://github.com/eddiejaoude/stargate"><img src="https://img.icons8.com/ios/24/000000/code-fork.png"></a> symbol at the top right corner.
+
+**2.** Clone the forked repository.
+
+```bash
+git clone https://github.com/<your-github-username>/stargate
+```
+
+**3.** Navigate to the project directory.
+
+```bash
+cd stargate
+```
+**5.** Create a new branch:
+```bash
+git checkout -b YourBranchName
+```
+
+**6.** Make changes in source code.
+
+**7.** Stage your changes and commit
+
+```bash
+git add .
+```
+```bash
+git commit -m "<your_commit_message>"
+```
+
+**8.** Push your local commits to the remote repo.
+
+```bash
+git push origin YourBranchName
+```
+
+**9.** Create a [PR](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request) !
+
+**10.** **Congratulations!** Sit and relax, you've made your contribution to [Stargate](https://github.com/eddiejaoude/stargate) project.
+
+***:trophy: After this, project leaders and mentors will review the changes and will merge your PR if they are found good, otherwise we will suggest the required changes.***
+
+## Style Guides for Git Commit Messages:memo:
+
+#### Here's a list of some good to have points, that can add more value to your contribution logs.
+
+- Use the present tense (example: "Add feature" and not "Added feature")
+- Use the imperative mood (example: "Move item to...", instead of "Moves item to...")
+- Limit the first line (also called subject line) to 50 characters or less
+- Capitalize the subject line
+- Separate subject from body with a blank line
+- Do not end the subject line with a period
+- Wrap the body at 72 characters
+- Use the body to explain what, why, vs, and how
+- Reference issues and pull requests liberally after the first line
+
+For more detailed reference to the above points, refer here: https://chris.beams.io/posts/git-commit.
+
+## 💥 Issues:
+For major changes, you are welcomed to open an issue  about what you would like to contribute. Enhancements will be appreciated.
+
+#### All the Best!🥇
+
+<p align = "center">
+
+<a href="https://github.com/muthuannamalai12"><img src="http://ForTheBadge.com/images/badges/built-by-developers.svg" alt="built by developers"></a>
```

Comment:
I think this image links to your profile, maybe we can change it to EddieHub or Eddie's github profile